### PR TITLE
add example for doom config

### PR DESCRIPTION
**Pull request by vmiheer**

### CODE REVIEW

**Comment by vmiheer** [comment]
Ideally it should be implemented as in https://github.com/emacs-evil/evil-collection (also see #15).

**Comment by psibi** [comment]
Thanks! Do you think you can probably also send a PR to evil-collection ?

File: README.org [hunk 1]
```diff
@@ -32,6 +32,15 @@ Optionally you can install it via [[https://github.com/quelpa/quelpa][quelpa]] t
   (justl-recipe-width 25))
 #+end_example
 
+For [[https://github.com/doomemacs/doomemacs][doom emacs]], you can install as:
```
Comment (psibi) [inline README.org:35]: Is this primarily for doom emacs or for integration with evil mode ?  I'm slightly confused as I have no idea about either of them. :-)

Comment (vmiheer) [reply README.org:35]: That particular change is only for doom emacs. I'll try to eventually (tm) create PR in evil-collection.

File: README.org [hunk 2]
```diff
@@ -32,6 +32,15 @@ Optionally you can install it via [[https://github.com/quelpa/quelpa][quelpa]] t
...
+
+#+begin_example elisp
+(use-package! justl
+  :config
+  (map! :n "e" 'justl-exec-recipe)
```
Comment (psibi) [inline README.org:40]:
```suggestion
  (map! :n "e" 'justl-exec-recipe))
```

Comment (psibi) [reply README.org:40]: And remove the parenthesis in the newline.

**Review by psibi — CHANGES_REQUESTED** [review]
Minor nitpick, LGTM apart from that.

File: README.org [hunk 3]
```diff
@@ -32,6 +32,15 @@ Optionally you can install it via [[https://github.com/quelpa/quelpa][quelpa]] t
   (justl-recipe-width 25))
 #+end_example
 
+For [[https://github.com/doomemacs/doomemacs][doom emacs]], you can install as:
+
+#+begin_example elisp
+(use-package! justl
+  :config
+  (map! :n "e" 'justl-exec-recipe))
+  )
```
Comment (psibi) [inline README.org:41]: I guess this line needs to be removed...

**Review by psibi — CHANGES_REQUESTED** [review]

File: README.org (hunk 3 above)
Comment (vmiheer) [reply README.org:41]: of mapping "e" to execute in evil's normal mode?

Comment (psibi) [reply README.org:41]: No, I mean there is an extra parenthesis at line no 41.

**Comment by psibi** [comment]
Thanks!

**Pull request merged**